draft-ietf-httpbis-priority.md

```diff
@@ -369,70 +380,82 @@ Stream ID:
 Priority Field Value:
 : The priority update value in ASCII text, encoded using Structured Headers.
 
-The HTTP/2 PRIORITY_UPDATE frame MUST NOT be sent prior to opening the
-stream.  If a PRIORITY_UPDATE is received prior to the stream being opened,
-it MAY be treated as a connection error of type PROTOCOL_ERROR.
+The PRIORITY_UPDATE frame MAY be sent before the stream that it references has
+been created. The Stream ID MUST be within the stream limit. If a server
```

Comment:
This is an important observation, thanks for catching it. I think this is a side effect of applying H3's rules naively rather than an intentional action. I'm open to phrasing this differently but I fear that the scope of the problem is actually a bit large. We probably need a little bit more thought about what it means to receive PRIORITY_UPDATES for H2 idle or closed streams. Especially given the amount of text RFC 7540 spends on describing this.

Perhaps the best action is to excise the entirety of this paragraph and create an issue to track this?

Reply:
I'd also be okay keeping it, provided there's an issue to figure out the right language in 7540-land.

Reply:
Split out as #1261.

Reply:
thanks! In that case I'm intending to keep the text